.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug report
+description: Report build and runtime bugs
+labels: ["Type: Bug"]
+body:
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
+      options:
+        - label: I have read the component documentation [ESP-IDF Components](https://components.espressif.com) and the issue is not addressed there.
+          required: true
+        - label: I am using target and esp-idf version as defined in component's idf_component.yml
+          required: true
+        - label: I have searched the [issue tracker](https://github.com/espressif/esp-usb/issues?q=is%3Aissue) for a similar issue and not found any related issue.
```

Comment:
This link is for `esp-usb` github issues

Reply:
espressif/tinyusb repo doesn't have the "Issues" itself. 

I asked to add them and changed the link accordingly. 
Thanks.